bol/ajax_service.php

```diff
@@ -586,14 +586,26 @@ public function findUsers( $kw, $limit = null )
             "method" => "MAILBOX_BOL_AjaxService::findUsers"
         ));
 
+        $params = array('kw' => $kw . '%');
+
+        if ( $kw !== null )
+        {
+            if ( $questionName == "username" )
+            {
+                $queryParts["where"] .= " AND base_user_table_alias.username LIKE :kw";
+            }
+            else
+            {
+                $params['questionName'] = $questionName;
+                $queryParts["where"] .= " AND qd.questionName=:questionName AND qd.textValue LIKE :kw";
+            }
+        }
+
         $query = 'SELECT DISTINCT qd.userId FROM ' . $questionDataTable . ' qd
```

Comment:
Use ow_bas_user table as main table and inner join questions table if displayName is not username